### DISCUSSION TITLE
How to set "Public" permissions for viewing docs or tables?

### DISCUSSION THREAD

**Discussion by aleph65** [AntlerVC/firetable#1501] [Support / Q&A] [answered]
My website needs to display images / data from Firestore. But the permissions seem to be managed / overwritten by Rowy. I can edit the permissions manually and set READ to TRUE for what I want. But it will be overwritten if I create a new table

What's the right way to set PUBLIC viewing for a specific Firebase table with Rowy?

Thanks!

**Comment by Chadha93** [comment, marked as answer]
Hi @aleph65, You can write your Table rules in your table settings along with Firestore security rules. To set your collection as public and only allow read for all (without auth) and allow write only to admin/owner/editors. You can follow this doc with example code snippets for each table rules and permissions - https://docs.rowy.io/how-to/roles?set-user-roles-tabs=user-management#example-collection-open-to-public.
